src/main/java/pl/tomaszdziurko/jvm_bloggers/metadata/MetadataKeys.java

```diff
@@ -0,0 +1,10 @@
+package pl.tomaszdziurko.jvm_bloggers.metadata;
+
+public enum MetadataKeys {
```

Comment:
I do not have strong opinion about this piece of code, so if you feel that if could be refactored to constants, fell free to do it.